packages/angular/.gitignore

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
is there a convention to naming the packages? let's have the folder name match the package name